WhirlyGlobeSrc/WhirlyGlobe-MaplyComponent/include/MaplyViewController.h

```diff
@@ -91,11 +97,8 @@
   */
 @interface MaplyViewController : MaplyBaseViewController
 
-/// @brief Initialize as a 3D map.
-- (id)init;
-
-/// @brief Initialize as a 2D map.
-- (id)initAsFlatMap;
+/// @brief Initialize as a flat or 3D map.
+- (id)initWithMapType:(MaplyMapType)mapType;
```

Comment:
In general, that would be good.  This will be 2.4.1, so we won't be breaking 2.4.